src/app/song/songs.component.html

```diff
@@ -0,0 +1,18 @@
+<div class="row mp-top-body">
+  <div class="row">
+    <div class="col-xs-12 text-center">
+      <h1 class="mp-title">Songs List</h1>
+    </div>
+  </div>
+
+  <div class="row">
+    <div class="col-xs-12">
+      <div class="mp-outer-block">
+      <div *ngFor="let img of images" class="mp-album-images">
```

Comment:
Change `let img of images` to let `let song of images` OR even better, instead of `images` name it `song` in the component.